src/scripts/main.js

```diff
@@ -1,3 +1,261 @@
 'use strict';
 
-// write code here
+const pageBody = document.querySelector('body');
+const tableHeader = document.querySelector('thead');
+const tableHeaderCells = tableHeader.querySelector('tr');
```

Comment:
The querySelector for 'tr' should be more specific to ensure it only selects the row within 'thead' and not any 'tr' element.